services/pong/src/service.js

```diff
@@ -116,4 +122,26 @@ export default class Service {
     // Update the ping details.
     this._pingStore.updateServicePing(service, online, ms);
   }
+
+  //Ping device and update how long the request takes
+  async _pingDevice(device, host) {
+    const ping = require('net-ping');
```

Comment:
The import should be at the top of the file, and use an ES6 import instead of `require(...)`

Reply:
fixed